supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -487,6 +487,17 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 
 *See also*:
 
+[discrete]
+[[codeready-linux-builder-repository]]
+==== image:images/caution.png[with caution] CodeReady Linux Builder (CRB) repository (noun)
+*Description*: In Red Hat Enterpise Linux, the _CodeReady Linux Builder repository_ provides additional packages for use by developers. Packages included in the CodeReady Linux Builder repository are unsupported. Note that you must use this term expanded and always mention that packages in this repository are unsupported (even in a release note, for example).
```

Comment:
```suggestion
*Description*: In Red Hat Enterpise Linux, the _CodeReady Linux Builder repository_ provides additional packages for use by developers. Packages included in the CodeReady Linux Builder repository are unsupported. Note that you must use this term expanded and always mention that packages in this repository are unsupported.
```
- Per IBM Style, avoid using parentheses in general text. In any case, consider omitting the final example phrase about the release notes because I don't think it's a significant enough item to include in this context.